prog/vm/host_nexus.rb

```diff
@@ -3,12 +3,16 @@
 class Prog::Vm::HostNexus < Prog::Base
   subject_is :sshable, :vm_host
 
-  def self.assemble(sshable_hostname, location_id: Location::HETZNER_FSN1_ID, family: "standard", net6: nil, ndp_needed: false, provider_name: nil, server_identifier: nil, spdk_version: Config.spdk_version, default_boot_images: [])
+  def self.assemble(sshable_hostname, location_id: Location::HETZNER_FSN1_ID, family: "standard", net6: nil, ndp_needed: false, provider_name: nil, server_identifier: nil, spdk_version: Config.spdk_version, vhost_block_backend_version: nil, default_boot_images: [])
     DB.transaction do
       unless Location[location_id]
         raise "No existing Location"
       end
 
+      if spdk_version && vhost_block_backend_version
+        raise "SPDK and VhostBlockBackend cannot be set simultaneously"
```

Comment:
I thought hosts could have some VMs running SPDK and other VMs running ubiblk. If that is correct, I don't think this case should error, and the prog should handle both being set. If incorrect, please ignore.